monitoring/metricsreporter/metrics_reporter.go

```diff
@@ -152,14 +148,71 @@ var (
 		Name: "ssv_message_validation_rsa_checks",
 		Help: "The amount message validations",
 	}, []string{})
+	pubsubPeerScore = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "ssv:p2p:pubsub:score:inspect",
+		Help: "Pubsub peer scores",
+	}, []string{"pid"})
+	pubsubPeerP4Score = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "ssv:p2p:pubsub:score:invalid_message_deliveries",
+		Help: "Pubsub peer P4 scores (for invalid message deliveries)",
```

Comment:
Maybe this is better:

```suggestion
                Help: "Pubsub peer P4 scores (sum of square of counters for invalid message deliveries)",
```

better do this in IDE cause I added spaces manually

Reply:
@GalRogozinski done, thx